tests/integration/annotation_import/test_mea_prediction_import.py

```diff
@@ -185,6 +186,29 @@ def test_create_from_local_file(tmp_path, model_run_with_data_rows,
     assert annotation_import.state == AnnotationImportState.FINISHED
     annotation_import_test_helpers.download_and_assert_status(
         annotation_import.status_file_url)
+    
+
+def test_predictions_with_custom_metrics(model_run,
```

Comment:
is this an additional test or is it something specific to the metrics use case?

Reply:
It checks whether a mea import with custom metrics on prediction level succeeds